[Bug]: Angular 14 support

### Version

11

### Steps to reproduce

Create brandnew Angular project via `ng new` with `--next` flag.

### Expected behavior

Nothing to report here...

### Actual behavior

At this point there is a dependency conflict:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: angular-net@1.0.0
npm ERR! Found: @angular-devkit/build-angular@14.0.0-next.11
npm ERR! node_modules/@angular-devkit/build-angular
npm ERR!   dev @angular-devkit/build-angular@"14.0.0-next.11" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular-devkit/build-angular@">=0.1002.4" from jest-preset-angular@11.1.1
npm ERR! node_modules/jest-preset-angular
npm ERR!   dev jest-preset-angular@"11.1.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/henry/.npm/eresolve-report.txt for a full report.
```

### Additional context

_No response_

### Environment

```shell
System: MacOS
Node: 14
```

--- a/package.json
+++ b/package.json
@@ -54,10 +54,10 @@
     "esbuild": ">=0.13.8"
   },
   "peerDependencies": {
-    "@angular-devkit/build-angular": ">=0.1102.19 <14.0.0",
-    "@angular/compiler-cli": ">=11.2.14 <14.0.0",
-    "@angular/core": ">=11.2.14 <14.0.0",
-    "@angular/platform-browser-dynamic": ">=11.2.14 <14.0.0",
+    "@angular-devkit/build-angular": ">=0.1102.19 <15.0.0",
+    "@angular/compiler-cli": ">=11.2.14 <15.0.0",
+    "@angular/core": ">=11.2.14 <15.0.0",
+    "@angular/platform-browser-dynamic": ">=11.2.14 <15.0.0",
     "jest": "^28.0.0",
     "typescript": ">=4.3"
   },

